docs/quickstarts/nextjs.mdx

```diff
@@ -118,68 +120,113 @@ The [`<ClerkProvider>`](/docs/components/clerk-provider) component provides acti
+Create a middleware.ts file. If your application uses the src/ directory, your middleware.ts file should be placed inside the src/ folder. If you are not using a src/ directory, place the middleware.ts file in your root directory alongside .env.local.
 
-Copy and paste the following code into your `middleware.ts`. With this, your entire application is protected.
+Copy and paste the following code into your middleware.ts. 
 
 ```tsx filename="middleware.ts"
-import { authMiddleware } from "@clerk/nextjs";
+import { clerkMiddleware } from "@clerk/nextjs/server";
 
-// This example protects all routes including api/trpc routes
-// Please edit this to allow other routes to be public as needed.
-// See https://clerk.com/docs/references/nextjs/auth-middleware for more information about configuring your Middleware
-export default authMiddleware({});
+export default clerkMiddleware()
 
 export const config = {
-  matcher: ["/((?!.+\\.[\\w]+$|_next).*)", "/", "/(api|trpc)(.*)"],
+  matcher: ['/((?!.+\\.[\\w]+$|_next).*)', '/', '/(api|trpc)(.*)'],
 };
 ```
 
-**Try accessing your app now by visiting [`http://localhost:3000`](http://localhost:3000).** The Middleware will redirect you to your Sign Up page, provided by Clerk's [Account Portal](/docs/account-portal/overview) feature. If you want to make other routes public, check out the [`authMiddleware` reference page](/docs/references/nextjs/auth-middleware). Go ahead and sign up to get access to your application. Your app is now fully protected by Clerk!
+### Use Clerk components to protect your content
 
-### Embed the `<UserButton />`
+Clerk offers a set of [prebuilt components](/docs/components/overview) that can gate access to content based on whether a user is signed in and has the correct permissions. You can use Clerk's [`<SignedIn>`](/docs/components/control/signed-in) and [`<SignedOut>`](/docs/components/control/signed-out) components to restrict access to content and even whole pages. The `<SignedIn>` component requires the user be signed in to view the content inside the tags. The `<SignedOut>` components requires the user to be signed out.
 
-Clerk offers a set of [prebuilt components](/docs/components/overview) to add functionality to your app with minimal effort. The [`<UserButton />`](/docs/components/user/user-button) allows users to manage their account information and log out, completing the full authentication circle.
+<CodeBlockTabs type="router" options={["App Router", "Pages Router"]}>
+```tsx filename="app/page.tsx" {1,6-11}
+import { SignedIn, SignedOut } from "@clerk/nextjs";
 
-Add the `<UserButton />` anywhere inside `<ClerkProvider>` in your app. First, add `import { UserButton } from "@clerk/nextjs";` to the top of your file. Then, add `<UserButton afterSignOutUrl="/"/>`. The `afterSignOutUrl` prop lets you customize what page the user will be redirected to after sign out.
+export default function Home() {
+  return (
+    <div className="h-screen">

```

Comment:
🤨 I am confused by this change

Reply:
Why change it to a fragment?

Reply:
This was **added** back into the repo (and thus should be in the guide since they should match line for line) by Lennart, since that's something from the Next app and its CSS.

Reply:
The other example uses a fragment, and there's no reason to have an empty div, especially not one with a random class, so I figured a fragment is best for consistency and semantics

Reply:
Its not a empty div. I thought we were a avoiding the classes too, but they got added back in. I'm going to change the Pages to use <div>s for now, so we are aligned there and can ship this. We can revisit this shortly when we don't have the time crunch, if you're good with that @jescalan?

Reply:
That's fine. I prefer a fragment to an unstyled div, since it reduces unnecessary wrappers in the output but this is not a strong enough to spend any more time on this, so if you prefer divs, for sure ship it 🚀